tests/routes/test_raster_tiles.py

```diff
@@ -14,25 +14,25 @@
 @pytest.mark.e2e
 @pytest.mark.parametrize("x, y, multiplier", [(0, 0, 1), (1, 1, 4)])
 def test_dynamic_tiles_no_params(x, y, multiplier, client):
-    """
-    Test dynamic tile cache
+    """Test dynamic tile cache.
+
     :param x: x block coordinate for tile
     :param y: y block coordinate for tile
     :param multiplier: the test tile has multiplier for each x, y block. This is used to check the expected result values.
     """
     try:
-        response = client.get(
+        with client.stream(
```

Comment:
Can you put a comment somewhere on why you needed to use `client.stream()` in these cases, where everywhere else, we always use `client.get()`?

Reply:
All i'm doing here is updating syntax to stream response from passing `stream=True` to `client.get` request to what I have updated in the recent versions of `httpx` library.  I lack the historical context at the moment for why streaming response was chosen in these tests, but I suspect it's to avoid filling up the memory of the test container with the image/tile responses.